projects/core/src/cart/cart-persistence.module.ts

```diff
@@ -24,7 +24,7 @@ export function uninitializeActiveCartMetaReducerFactory(): MetaReducer<any> {
   const metaReducer = (reducer: ActionReducer<any>) => (state, action) => {
     const newState = { ...state };
     if (action.type === '@ngrx/store/init') {
-      newState.cart = { ...newState.cart, ...{ active: undefined } };
+      newState.cart = { ...newState.cart, ...{ active: null } };
```

Comment:
Why we needed to change it from undefined to null?

Reply:
1. Problem with `undefined` value was the reducer which have default value, thus it was not working as the undefined was instantly replaced with the default `''`. That's why we changed to null.